src/biopack/psg/CompSens.h

```diff
@@ -46,7 +47,7 @@ int    xni, xni2, xni3;
     psg_abort(1);
   }
 
-  fgets(cmd, 512, src);
+  ret = fgets(cmd, 512, src);
```

Comment:
the 512 could be sizeof(cmd)

Reply:
Agreed. For this PR, I tried to make only the changes needed to silence the warnings. While reviewing the code I found a number of other infelicities in the code but I resisted making changes. A fairly serious one is a potential endless loop in sglCommon.c (lines 6715-6718)